mcfix_account/static/src/js/reconciliation/reconciliation_model.js

```diff
@@ -27,22 +27,5 @@ odoo.define("account.ReconciliationModel.multicompany", function(require) {
         }
       });
     },
-    load: function(context) {
-      var self = this;
-      var _super = self._super.bind(self);
-      var statement_ids = context.statement_ids;
-      return self
-        ._rpc({
-          model: "account.bank.statement",
```

Comment:
function does not exist anymore, so it is failing. But, Odoo currently does it correctly

Reply:
I deleted the method `get_company_ids` during the migration to v13. I didn't check it was used here. But if you checked that Odoo does it correctly, then it ok for me to delete all this part.